plugins/wp-graphql-headless-webhooks/src/Autoloader.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Includes the composer Autoloader used for packages and classes in the src/ directory.
+ *
+ * @package WPGraphQL\Webhooks
+ * @since 0.1.4
+ */
+
+declare( strict_types = 1 );
+
+namespace WPGraphQL\Webhooks;
+
+/**
+ * Class - Autoloader
+ *
+ * @internal
+ */
+class Autoloader {
```

Comment:
That would be great for the team to discuss and find the common solution/way for the autoloading for our new plugins. And use unified for all new plugins.